### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by wilsonsj100**
Adding examples for whattheflux package.

### CODE REVIEW

File: CH4 Example/Tree_CH4_Uptake_Example.R [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#Last Updated: 2024-03-21
+
+
+#CH4 data from COMPASS Experiment (Source: Stephanie J. Wilson & Samuel Wright) 
+#Data collected from tree chambers at the Sweet Hall Forested Swamp
+################################################################################################################
+
+###########################################  CH4 Fluxes ########################################################
+#Set up
+setwd("S:/Biogeochemistry/People/Wilson (Steph)/Manuscripts/Whattheflux Tech Note/Tree CH4 Uptake Example")
```
Comment (bpbond) [inline CH4 Example/Tree_CH4_Uptake_Example.R:15]: @wilsonsj100 Remove this -- it's only valid for your machine

File: CH4 Example/READ ME
Comment (bpbond) [inline CH4 Example/READ ME:1]: Please rename to `README.md` to follow package conventions

**Review by bpbond — CHANGES_REQUESTED** [review]
A few changes and all tests are failing. I would suggest make a single directory `examples/` and then put the CH4 and N2O folders within that.

**Comment by wilsonsj100** [comment]
I just deleted both the folders I made for now.

**Comment by bpbond** [comment]
Let's come back to this!

**Pull request closed without merging**